src/main/scala/com/datastax/gatling/plugin/response/DseResponseHandler.scala

```diff
@@ -153,8 +155,26 @@ class CqlResponseHandler[T <: Statement[T], B <: StatementBuilder[B,T]](val next
                          val metricsLogger: MetricsLogger)
   extends DseResponseHandler[T, AsyncResultSet, CqlResponse] {
   override protected def tag: String = dseAttributes.tag
-  override protected def queries: Seq[String] = Seq.empty
+  override protected def queries: Seq[String] = formatStatement(stmt)
   override protected def specificChecks: List[DseCqlCheck] = dseAttributes.cqlChecks
   override protected def newResponse(rs: AsyncResultSet): CqlResponse = new CqlResponse(rs, dseAttributes)
   override protected def coordinator(rs: AsyncResultSet): Node = rs.getExecutionInfo.getCoordinator
+
+  def formatSimpleStatement(s:SimpleStatement):String = s.getQuery
+
+  def formatBoundStatement(s:BoundStatement):String = s.getPreparedStatement.getQuery
+
+  def formatStatement(stmt:Statement[T]):Seq[String] = {
```

Comment:
I think that you need to add some formatter plugin to your project. For example according to scala coding convention, after the variable definition, you need to have white space, so this:
`def formatStatement(stmt:Statement[T]):Seq[String]` should become this:
`def formatStatement(stmt: Statement[T]): Seq[String]`
This is the same for a lot of places.

Or:
```
  val createTable = SchemaBuilder.createTable(testKeyspace, table_name)
    .ifNotExists
    .withPartitionKey("id",DataTypes.INT)
    .withClusteringColumn("str", DataTypes.TEXT)
    .build
```
should be:
```
  val createTable = SchemaBuilder.createTable(testKeyspace, table_name)
    .ifNotExists
    .withPartitionKey("id", DataTypes.INT)
    .withClusteringColumn("str", DataTypes.TEXT)
    .build
```
(space after every argument to be consistent)

Reply:
There is a scalafmt file defined in this project but it's conventions aren't universally followed throughout the code base.  As a result when I tried to apply a global change to match what was specified there I got a number of format-related changes that aren't related to my changes at all.  In order to keep this PR focused on changes relevant only to the driver upgrade I decided to avoid worrying about that for now.

Instead, I'd argue for a separate PR following this PR which converts the entire code base to something matching the scalafmt definition.